drop/installRPackages.R

```diff
@@ -1,37 +1,42 @@
 options(repos=structure(c(CRAN="https://cloud.r-project.org")), warn = -1)
-suppressPackageStartupMessages(library(data.table))
 
 if (!requireNamespace('BiocManager', quietly = TRUE)) {
     install.packages('BiocManager')
     BiocManager::install("remotes")
 }
+if (!requireNamespace('data.table', quietly = TRUE)) {
+    install.packages('data.table')
+}
 
+suppressPackageStartupMessages(library(data.table))
+
+# do not turn wanrings into errors. E.g. "Package XXX build for R 4.0.X"
+Sys.setenv(R_REMOTES_NO_ERRORS_FROM_WARNINGS="true")
 
 args <- commandArgs(trailingOnly=TRUE)
-packages <- fread(args[1], fill = TRUE)
-packages <- packages[!startsWith(package, "#")]
+if (file.exists(args[1])){
+    packages <- fread(args[1], fill = TRUE)
+} else {
+    packages <- data.table(
+        package=gsub("=.*", "", unlist(args)),
+        version=gsub(".*=", "", unlist(args)))
+    packages[package == version, version:=NA]
+}
 installed <- as.data.table(installed.packages())
 
 for (pckg_name in packages$package) {
     package_dt <- packages[package == pckg_name]
-    pckg_name <- tail(unlist(strsplit(pckg_name, split = "/")), n = 1)
+    pckg_name <- gsub(".*/", "", pckg_name)
     version <- package_dt$version
     
-    if (pckg_name %in% installed$Package &
-      (version == "" || installed[Package == pckg_name, Version] == version)
-    ) {
-        #message(paste(pckg_name, "already installed"))
-    } else {
-        if (package_dt$bioconductor == TRUE) {
-            INSTALL <- BiocManager::install
-        } else {
-            INSTALL <- install.packages
-        }
+    if (!pckg_name %in% installed$Package || (!is.na(version) && compareVersion(
+            installed[Package == pckg_name, Version], version) < 0)) {
```

Comment:
alternatively, we could use `remotes::install_version` or `devtools::install_version` to ensure that the correct version of github packages are installed.

Reply:
I would not pin the version but rather have them lower bounded. Especially the user is free to use any R version and hence it could conflict with bioconductor. Hence just lower bound.